package.json

```diff
@@ -18,15 +18,15 @@
     "test": "tsdx test",
     "test:watch": "tsdx test --watch",
     "test:e2e": "tsdx test --config jest-e2e-config.json",
-    "lint": "tsdx lint --fix",
-    "prepare": "tsdx build",
+    "lint": "eslint --fix src",
```

Comment:
Do you want to have ci check that the code was linted correctly? Like a lint-check command?

Reply:
i think that would be useful, i could add that

Reply:
i added `lint-check`